lib/resources/r4/document_reference.yaml

```diff
@@ -230,6 +243,7 @@ fields:
   type: dateTime
   action:
     - create
+    - update
   description: The date that the attachment was first created.
   url: https://www.hl7.org/fhir/datatypes-definitions.html#Attachment.creation
   note: A time component must be provided with the date.
```

Comment:
Can we add a note here about how the new creation date must not be earlier than the previous creation date.  We'll need to have a separate section for writes and updates.  Under the note section for the update action, we'll need to add:

```
  note: A time component must be provided with the date.  The new date must not be earlier than the previous content.attachment.creation date.
```